core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -78,6 +81,18 @@ abstract protected[controller] class RestAPIVersion(
             "build" -> build.toJson,
             "buildno" -> buildno.toJson)
     }
+	
+	/**
+     * Information which describes details of the system metadata.
+     */
+    def systemInfo = {
+        JsObject(
+            "name" -> "openwhisk".toJson,
+            "support" -> "https://github.com/openwhisk/openwhisk/issues".toJson,
```

Comment:
Do we want to hardcode this? Other deployments might want to have a different URL?